### PR TITLE
Kafka relay support

### PR DESCRIPTION
**Pull request by blinktag**
* Kafka relay support
* Flags are no longer being duplicated in relay mode

### CODE REVIEW

File: backends/kafka/relay.go [hunk 1]
```diff
@@ -1,7 +1,111 @@
...
+		GRPCAddress: opts.RelayGRPCAddress,
+		NumWorkers:  opts.RelayNumWorkers,
+		Timeout:     opts.RelayGRPCTimeout,
+		RelayCh:     make(chan interface{}, 1),
+		DisableTLS:  opts.RelayGRPCDisableTLS,
+	}
+
+	grpcRelayer, err := relay.New(relayCfg)
+	if err != nil {
+		return errors.Wrap(err, "unable to create new gRPC replayer")
```
Comment (dselans) [inline backends/kafka/relay.go:52]:
```suggestion
		return errors.Wrap(err, "unable to create new gRPC relayer")
```

File: cli/kafka.go [hunk 2]
```diff
@@ -39,17 +40,48 @@ func HandleKafkaFlags(readCmd, writeCmd *kingpin.CmdClause, opts *Options) {
...
+
+	if relayType != "" {
+		rec = relayCmd
+	} else {
+		rec = relayCmd.Command("kafka", "Kafka message system")
+	}
+
+	addSharedKafkaFlags(rec, opts)
+	addReadKafkaFlags(rec, opts)
+	//addKafkaRelayFlags(rec, opts)
```
Comment (dselans) [inline cli/kafka.go:57]: What's this?

File: cli/kafka.go [hunk 3]
```diff
@@ -61,3 +93,7 @@ func addReadKafkaFlags(cmd *kingpin.CmdClause, opts *Options) {
 func addWriteKafkaFlags(cmd *kingpin.CmdClause, opts *Options) {
 	cmd.Flag("key", "Key to write to kafka (not required)").StringVar(&opts.Kafka.WriteKey)
 }
+
+//func addKafkaRelayFlags(cmd *kingpin.CmdClause, opts *Options) {
```
Comment (dselans) [inline cli/kafka.go:97]: Can get rid of this.

**Review by dselans — APPROVED** [review]
Minor notes. Nice!

File: cli/kafka.go (hunk 3 above)
Comment (blinktag) [reply cli/kafka.go]: Removed!

File: cli/kafka.go (hunk 2 above)
Comment (blinktag) [reply cli/kafka.go]: Removed!

File: backends/kafka/relay.go (hunk 1 above)
Comment (blinktag) [reply backends/kafka/relay.go]: Fixed!

**Pull request merged**